.github/workflows/deploy-stg-ecs.yml

```diff
@@ -3,6 +3,7 @@ on:
     branches:
       - "master"
       - "integration"
+      - "DOP-2902"
```

Comment:
We'll want to drop this before merging - just flagging, sure you're on top of it 😄